backend/arkham-core/library/Arkham/Asset/Cards/BrandOfCthugha1.hs

```diff
@@ -0,0 +1,45 @@
+module Arkham.Asset.Cards.BrandOfCthugha1 (brandOfCthugha1, BrandOfCthugha1 (..)) where
+
+import Arkham.Ability
+import Arkham.Asset.Cards qualified as Cards
+import Arkham.Asset.Runner
+import Arkham.Prelude
+
+newtype BrandOfCthugha1 = BrandOfCthugha1 AssetAttrs
+  deriving anyclass (IsAsset, HasModifiersFor)
+  deriving newtype (Show, Eq, ToJSON, FromJSON, Entity)
+
+brandOfCthugha1 :: AssetCard BrandOfCthugha1
+brandOfCthugha1 = asset BrandOfCthugha1 Cards.brandOfCthugha1
+
+instance HasAbilities BrandOfCthugha1 where
+  getAbilities (BrandOfCthugha1 a) =
+    [ restrictedAbility a 1 ControlsThis $ fightAction_ ]
+
+instance RunMessage BrandOfCthugha1 where
+  runMessage msg a@(BrandOfCthugha1 attrs) = case msg of
+    UseThisAbility iid (isSource attrs -> True) 1 -> do
+      player <- getPlayer iid
+      push 
+        $ chooseOne 
+          player 
+          [ SkillLabel 
+            sType 
+            [ skillTestModifier (toAbilitySource attrs 1) iid (SkillModifier sType 1)
+            , chooseFightEnemy iid (toAbilitySource attrs 1) sType
+            ]
+          | sType <- [#willpower, #combat]
+          ]
+      pure a
+    PassedThisSkillTestBy iid (isAbilitySource attrs 1 -> True) n -> do
+      player <- getPlayer iid
+      pushWhen (n == 0) $ LoseActions iid (toAbilitySource attrs 1) 1
+      push $ chooseAmounts player "Amount of Charges to Spend" (MaxAmountTarget 2) [("Charges", (1, 2))] attrs
```

Comment:
We'd want to make sure you have the charges to spend, so to run this message we want to ensure there is at least one charge, and set a max equal to the minimum number between 2 and the remaining charge count.